### ISSUE TITLE
Investigate: bulleted lists in dev docs are broken

### ISSUE THREAD

**Issue by indirectlylit** [learningequality/kolibri#7958]
## Acceptance Criteria

- [x] Determine the proper path forward for fixing it.
- [x] Either: Create a new issue to fix the problem once a likely solution is found OR fix it if it's a quick fix and submit a PR that closes this Issue.

## Observed behavior

All bulleted lists in the dev docs are broken:

> ![image](https://user-images.githubusercontent.com/2367265/113760728-70354980-96cb-11eb-80f2-e24803a2b736.png)

## Expected behavior

Bulleted lists should be properly indented and annotated

## User-facing consequences

Harder to read dev docs

## Steps to reproduce

Navigate dev docs at https://kolibri-dev.readthedocs.io/

## Context

This is definitely a regression, but after some initial investigation it seems to have existed for some time now – at least since 0.13

**Comment by radinamatic** [comment]
I'm not able to replicate this when I build the dev docs locally on current `develop`, bulleted lists display OK 🤔

![LEDev2104 (start)  Running  - Oracle VM VirtualBox_034](https://user-images.githubusercontent.com/1457929/113967934-64b06280-9832-11eb-9c04-0e0c55ea86e4.png)

I checked out the docs versions backward from `develop` to `0.12.x` and was not able to replicate it anywhere. I'll investigate if it might be the RTD platform issue.

**Comment by chrislun16** [comment]
Is the issue that this isn't in a bulleted list (the section about a rule of thumb for inline bidirectional text)?
<img width="1272" alt="Screen Shot 2021-04-12 at 10 39 58 AM" src="https://user-images.githubusercontent.com/24814507/114437594-9b141780-9b7b-11eb-9d5b-7b41ccc379d1.png">


I built it locally and got the above screenshot. Looking at the i18n.html file, we are using * for a bulletpoint, so I'm not sure where the issue could be. Any suggestions?

**Comment by indirectlylit** [comment]
Looks like it's some kind of CSS problem because there are `<ul>` and `<li>` elements rendered in the DOM:

![image](https://user-images.githubusercontent.com/2367265/114444054-496f8b00-9b83-11eb-9c9f-8fc2c43a2d57.png)

**Comment by nucleogenesis** [comment]
I get this issue too in Chrome. Another example: https://kolibri-dev.readthedocs.io/en/develop/getting_started.html#manual-testing

But [the same on Github](https://github.com/learningequality/kolibri/blob/develop/docs/getting_started.rst) shows the bullets as expected.

The `https://kolibri-dev.readthedocs.io/en/develop/_static/css/theme.css` file declares:

```css
ul, ol, dl {
    margin: 0;
    padding: 0;
    list-style: none;
    list-style-image: none;
}
```

Which strips the bullets. I don't know if we own that file - but if we do we should fix it there.

**Comment by jonboiser** [comment]
There's this file https://github.com/learningequality/kolibri/blob/7f76b3b2caa9b7866b318c7140d3a34704e3962d/docs/_build/_static/theme_overrides.css#L1

where I think you can add a new rule to override the `list-style: none` rule in the default styles.

**Comment by jonboiser** [comment]
@radinamatic I think by commenting out the `list-style` overrides in https://github.com/jonboiser/kolibri/pull/7, this fixes this issue.

![image](https://user-images.githubusercontent.com/10248067/114591160-72a12180-9c3e-11eb-8218-090c0f620200.png)

I wouldn't have thought it since the rules in the override CSS are so specific 🤷

**Comment by radinamatic** [comment]
OK, [I approved your PR](https://github.com/learningequality/kolibri/pull/7984#pullrequestreview-634842641) @jonboiser, let's see if commenting out those rules will solve this! 🤞🏽

**Comment by jonboiser** [comment]
Nope, the latest build still has the missing bullet points. It seems that `theme.css` is different on  https://kolibri-dev.readthedocs.io/ than when we're building locally, since the `ul`s look fine locally.

**Comment by jonboiser** [comment]
The logs here might be useful for debugging why things are different online vs. running `make docs`

https://readthedocs.org/api/v2/build/13501819.txt

**Comment by radinamatic** [comment]
So annoying... 😒 

We pin the [`sphinx-rtd-theme` for dev docs to 0.4.3](https://github.com/learningequality/kolibri/blob/develop/requirements/docs.txt#L6), which is 2 years old, I'll try updating that!

**Comment by jonboiser** [comment]
Fixed in #8002 🎉

**Comment by indirectlylit** [comment]
thanks!
